astropy/uncertainty/distributions.py

```diff
@@ -90,8 +92,9 @@ def poisson(center, n_samples, cls=Distribution, **kwargs):
 
     Returns
     -------
-    distr : ``cls``, usually `Distribution`
-        The sampled poisson distribution.
+    distr : `Distribution` or object
```

Comment:
Should this be `~astropy.uncertainty.Distribution`?

Reply:
✅